.github/actions/setup-e2e-env/action.yml

```
@@ -184,10 +194,13 @@ runs:
 
     ## Yarn Setup & Cache Management
 
-    # - name: Corepack
-    #   id: corepack
-    #   run: corepack enable && corepack prepare yarn@${{ inputs.yarn-version }} --activate
-    #   shell: bash
+    - name: Get Corepack install command
+      id: get-corepack-command
+      env:
+        YARN_VERSION: ${{ inputs.yarn-version }}
+      shell: bash
+      run: |
+        echo "COREPACK_COMMAND=corepack enable && corepack prepare yarn@$YARN_VERSION" >> "$GITHUB_OUTPUT"
```

Nit: should this be ?
```suggestion
        echo "COREPACK_COMMAND=corepack enable && corepack prepare yarn@$YARN_VERSION --activate" >> "$GITHUB_OUTPUT"